### DISCUSSION TITLE
Failed to import full mesh

### DISCUSSION THREAD

**Discussion by zephyroal** [eliemichel/MapsModelsImporter#190] [Q&A]
hi there, 

it's really amazing work!

after I tried almost every release version, everything seem work fine, 
but I only got a flat upset mesh:
![image](https://user-images.githubusercontent.com/2037885/138705640-6813cb60-1f27-4717-8858-cd9d760fe64c.png)

here is one rdc:
https://drive.google.com/file/d/1tDLdM_AnlzkAdP-BUDrj1eStDSCVP0Il/view?usp=sharing

thx a lot！

**Comment by eliemichel** [comment]
Don't you have the expected geometry if you zoom a lot around the origin? The source of this is that you load a capture taken such that it includes huge chunks of Earth, for the background.
